suites/tipset/miner_flow.go

```diff
@@ -234,9 +240,11 @@ func TestMinerSubmitFallbackPoSt(t *testing.T, factory state.Factories) {
 		return ownerCallSeq
 	}
 
-	candidates := []abi_spec.PoStCandidate{{
+	candidates := []abi_spec.SectorInfo{{
 		RegisteredProof: abi_spec.RegisteredProof_StackedDRG32GiBPoSt,
-		ChallengeIndex:  0,
+		SectorNumber:    abi_spec.SectorNumber(0),
+		// XXX: probably wrong
+		SealedCID: cid.Undef,
```

Comment:
This should come from the proven committed sector created above.

This method generally is all wrong now. You need to work out which deadline the sector is due at and post the proof at the right time. (Don't hardcode any numbers here, the sector->deadline allocation  will change).